packages/fast-components-react-msft/src/subheading/subheading.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { get } from "lodash-es";
+import { Foundation, HandledProps, TypeLevel, TypographyTag  } from "@microsoft/fast-components-react-base";
+import {
+    ISubheadingHandledProps,
+    ISubheadingManagedClasses,
+    ISubheadingUnhandledProps,
+    SubheadingLevel
+} from "./subheading.props";
+
+import Typography from "../typography";
+import { IManagedClasses, ISubheadingClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Subheading extends Foundation<ISubheadingHandledProps & IManagedClasses<ISubheadingClassNameContract>, React.HTMLAttributes<HTMLElement>, {}> {
+    public static defaultProps: Partial<ISubheadingHandledProps> = {
+        level: SubheadingLevel._1
+    };
+
+    protected handledProps: HandledProps<ISubheadingHandledProps & IManagedClasses<ISubheadingClassNameContract>> = {
+        level: void 0,
+        managedClasses: void 0,
+        tag: void 0
+    };
+
+    private get tag(): TypographyTag {
+        return this.props.tag ? TypographyTag[this.props.tag] : TypographyTag.h3;

```

Comment:
Instead of applying a default of `TypographyTag.h3` here, can you instead add a `defaultProp` instead? You can then change th render method to `tag={TypographyTag[this.props.tag]}` and remove this getter entirely.

Reply:
I think the getter is still required since it's mapping to another enum, right? But I did add the default prop which makes the getter a bit cleaner.

Reply:
done and done